Core/COEditingContext.m

```diff
@@ -369,6 +392,11 @@ - (void)undeletePersistentRoot: (COPersistentRoot *)aPersistentRoot
  */
 - (void)updateDeadRelationshipCacheForUndeletedPersistentRoot: (COPersistentRoot *)aPersistentRoot
 {
+	// FIXME:
```

Comment:
It makes sense to comment it out with #if 0, I would expect -updateDeadRelationshipCacheForUndeletedPersistentRoot: to break some tests.

I'll try to move the code to -updateCrossPersistentRootReferencesToPersistentRoot:branch:isDeleted: and see how it goes.